packages/contracts-bedrock/contracts/deployment/BaseSystemDictator.sol

```diff
@@ -2,7 +2,7 @@
 pragma solidity 0.8.15;
 
 import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
-import { SystemConfig } from "./SystemConfig.sol";
+import { SystemConfig } from "./DeployConfig.sol";
```

Comment:
I renamed the file `deployment/SystemConfig.sol` so that there are not 2 files with the same name